db/sql_changeset_schema

```diff
@@ -36,8 +36,11 @@ ALTER TABLE user_accounts ADD CONSTRAINT email_uniq UNIQUE (email);
 --- alter table user_files add column zipfilesize bigint;
 --- update user_files set zipfilesize = length(data);
 CREATE INDEX user_account_devices_userid_idx on user_account_devices(userid);
+CREATE INDEX user_files_name_idx on user_files(name);

```

Comment:
Too many indexes doesn't improve situation usually...

Reply:
It is not too many, it's just enough to speed up our WHERE/MAX/ORDER BY/GROUP BY query elements.